store/postgres/src/relational_queries.rs

```diff
@@ -1971,6 +2006,54 @@ impl<'a> Query for FindQuery<'a> {
 
 impl<'a, Conn> RunQueryDsl<Conn> for FindQuery<'a> {}
 
+#[derive(Debug, Clone)]
+pub struct FindRangeQuery<'a> {
+    table: &'a Table,
+    eb_range: EntityBlockRange,
+}
+
+impl<'a> FindRangeQuery<'a> {
+    pub fn new(table: &'a Table, block_range: Range<BlockNumber>) -> Self {
+        let eb_range = EntityBlockRange::new(table.immutable, block_range);
+        Self { table, eb_range }
+    }
+}
+
+impl<'a> QueryFragment<Pg> for FindRangeQuery<'a> {
+    fn walk_ast<'b>(&'b self, mut out: AstPass<'_, 'b, Pg>) -> QueryResult<()> {
+        out.unsafe_to_cache_prepared();
+
+        // Generate
+        //    select '..' as entity, to_jsonb(e.*) as data
+        //      from schema.table e where id = $1
+        out.push_sql("select ");
+        out.push_bind_param::<Text, _>(self.table.object.as_str())?;
+        out.push_sql(" as entity, to_jsonb(e.*) as data\n");
+        out.push_sql("  from ");
+        out.push_sql(self.table.qualified_name.as_str());
+        out.push_sql(" e\n where ");
+        // TODO: do we need to care about it?
```

Comment:
Yes, we do. Enabled in later [PR](https://github.com/graphprotocol/graph-node/pull/5614).